app/models/graphql/transactions_query.rb

```diff
@@ -1,4 +1,6 @@
-Graphql::TransactionsQuery = ShopifyPartnerAPI.client.parse <<~'GRAPHQL'
+require 'shopify_partner_api'
```

Comment:
**[standard]** <Style/StringLiterals> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.
```suggestion
require "shopify_partner_api"
```